Jenkinsfile

```diff
@@ -36,17 +36,20 @@ pipeline {
         stage('Verify') {
             steps {
                 sh 'pip install bandit'
-                sh 'bandit -lll -s B303 -r .'
+                sh 'bandit -lll -s B303 -r . -o "reports/bandit.txt"'
             }
         }
         stage('Deploy') {
             steps {
-                sh 'echo Deploy'
+                sh 'tar -cvzf build.tar.gz build/'
+                sh 'tar -cvzf reports.tar.gz reports/'
```

Comment:
Doing the tar packaging of the *reports* folder on the *Deploy* stage won't garantee that you have the reports if another previus task fails.
I think it will be better to make the tar on the *post* job.